api/v1/routes/iterations.rb

```diff
@@ -74,13 +74,13 @@ class Iterations < Core
           data['path'] = segments[2..-1].join("/")
         end
 
-        track = Trackler.tracks[data['language']]
+        track = Trackler.tracks[data['language'].downcase]
 
         unless track.exists?
           halt 400, { error: "Unknown language track %s" % data['language'] }.to_json
         end
 
-        unless track.implementations[data['problem'].to_s].exists?
+        unless track.implementations[data['problem'].downcase].exists?
```

Comment:
Yes, please. Don't worry about adding a test for that case, though. I'm just going to live with it being a thing that shouldn't happen, but might.